yukyungdeok/6week/2750.py

```diff
@@ -0,0 +1,8 @@
+# 2750번 수 계산하기
+n = int(input())
+li = []
+for _ in range(n):
+    li.append(int(input()))
+li = sorted(li)
+for i in range(len(li)):
```

Comment:
```suggestion
for i in li
```
으로 하면 i에 리스트의 원소가 담기므로 print(i)를 출력하면 간단해짐